docs/concepts-messaging.md

```diff
@@ -53,153 +53,6 @@ The default size of a message is 5 MB. You can configure the max size of a messa
-
-:::
-
-#### Enable Message Chunking
-
-**Prerequisite:** Disable batching by setting the `enableBatching` parameter to `false`.
-
-The message chunking feature is OFF by default.
-To enable message chunking, set the `chunkingEnabled` parameter to `true` when creating a producer.
-
-:::note
-
-If the consumer fails to receive all chunks of a message within a specified period, it expires incomplete chunks. The default value is 1 minute. For more information about the `expireTimeOfIncompleteChunkedMessage` parameter, refer to [org.apache.pulsar.client.api](/api/client/).
-
-:::
-
-## Consumers
-
-A consumer is a process that attaches to a topic via a subscription and then receives messages.
-
-![Consumer](/assets/consumer.svg)
-
-A consumer sends a [flow permit request](developing-binary-protocol.md#flow-control) to a broker to get messages. There is a queue at the consumer side to receive messages pushed from the broker. You can configure the queue size with the [`receiverQueueSize`](client-libraries-java.md#configure-consumer) parameter. The default size is `1000`). Each time `consumer.receive()` is called, a message is dequeued from the buffer.
-
-### Receive modes
-
-Messages are received from [brokers](reference-terminology.md#broker) either synchronously (sync) or asynchronously (async).
-
-| Mode          | Description                                                                                                                                                                                                   |
-|:--------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Sync receive  | A sync receive is blocked until a message is available.                                                                                                                                                  |
-| Async receive | An async receive returns immediately with a future value—for example, a [`CompletableFuture`](http://www.baeldung.com/java-completablefuture) in Java—that completes once a new message is available. |
-
-### Listeners
-
-Client libraries provide listener implementation for consumers. For example, the [Java client](client-libraries-java.md) provides a {@inject: javadoc:MesssageListener:/client/org/apache/pulsar/client/api/MessageListener} interface. In this interface, the `received` method is called whenever a new message is received.
-
 ### Acknowledgment
 
 The consumer sends an acknowledgment request to the broker after it consumes a message successfully. Then, this consumed message will be permanently stored, and deleted only after all the subscriptions have acknowledged it. If you want to store the messages that have been acknowledged by a consumer, you need to configure the [message retention policy](concepts-messaging.md#message-retention-and-expiry).
```

Comment:
We should not move these parts `Producers` and `Consumers`. They are very basic concepts of the Pulsar messaging. . For example, this section is dependent on `consumer`. It will make users confused if we remove them.

Reply:
Discussed with @RobertIndie, the beginning paragraph of the Messaging concept topic has introduced both producer and consumer. So this move is okay.